### DISCUSSION TITLE
part-ii/2016-67/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#321] [Question comments]
Part II 2016 Galois Theory: Paper 4, Section II, H 

https://questions.tripos.org/part-ii/2016-67/

**Comment by DannyDuberstein** [comment]
(a) Hopefully can just quote that L/Q is not solvable. If Q(a)/Q contained in a radical extnesion then f would have a linear factor over that field, and could solve the rest by radicals as degree <= 4. 

(b) No, G is in S_26. 
27 = 3^3 - 1 so mod 3 f factors into (all of the) irreducibles degree 3 conclude by dedekind
